net/rootd/src/rootd.cxx

```diff
@@ -465,96 +442,198 @@ static const char *HomeDirectory(const char *name)
 }
 
 //______________________________________________________________________________
-char *RootdExpandPathName(const char *name)
+static const char *WorkingDirectory()
 {
-   // Expand a pathname getting rid of special shell characters like ~.$, etc.
-   // Returned string must be freed by caller.
+   // Returns the current working directory.
 
-   const char *patbuf = name;
+   static char path[kMAXPATHLEN];
 
-   // skip leading blanks
-   while (*patbuf == ' ')
-      patbuf++;
-
-   // any shell meta characters?
-   for (const char *p = patbuf; *p; p++)
-      if (strchr(shellMeta, *p))
-         goto needshell;
+   if (getcwd(path, kMAXPATHLEN)) return path;
+   return 0;
+}
 
-   return strdup(name);
+//______________________________________________________________________________
+Bool_t RootdExpandFileName(const char *fname, char *xname, const int kBufSize)
```

Comment:
Is this routine local to this file?  Shouldn't it be static and/or unnamed-namespace?